lib/ASTGen/Sources/ASTGen/Exprs.swift

```diff
@@ -289,6 +308,63 @@ extension ASTGenVisitor {
     )
   }
 
+  func generateArgumentList(leftParen: TokenSyntax?, labeledExprList: LabeledExprListSyntax, rightParen: TokenSyntax?, trailingClosure: ClosureExprSyntax?, additionalTrailingClosures: MultipleTrailingClosureElementListSyntax?) -> BridgedArgumentList {
```

Comment:
Worth running swift-format? Not sure how much has changed since we last ran it

Reply:
There're some suggested changes from swift-format, but for this PR, I only applied it to the changes in this PR.